packages/components/app/styles/components/table.scss

```diff
@@ -16,32 +17,53 @@
 
 .hds-table--striped {
   tbody {
-    tr:nth-child(odd) {
+    tr:nth-child(even) {
       background-color: var(--token-color-surface-faint);
     }
   }
 }
 
+.hds-table--density-short td {
+  padding: 4px 16px;
+}
+
+
+.hds-table--density-medium td {
+  padding: 12px 16px;
+}
+
+.hds-table--density-tall td {
+  padding: 20px 16px;
+}
+
 .hds-table__thead {
   border-bottom: 1px solid var(--token-color-border-primary);
 
-  th {
+  .hds-table__th,
+  .hds-table__th-sort {
+    height: 48px;
     padding: 8px 16px;
     color: var(--token-color-foreground-strong);
     font-weight: var(--token-typography-font-weight-semibold);
+    font-size: 14px;
```

Comment:
I think I have already mentioned elsewhere: we should use the CSS helper classes in the templates (or design tokens in the CSS) instead of these values (in Figma I see they've used standard text styles)

Reply:
Let's work on this in the PR related to styles (https://github.com/hashicorp/design-system/pull/658)